templates/home.html

```diff
@@ -63,4 +73,25 @@ <h6>{{ tweet.original_tweet.user.first_name }}</h6>
     </ul>
   {% endif %}
 </div>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+
+{% block specific_js %}
+  <script>
+    function deleteTweet(id) {
```

Comment:
Make it a common function, and pass URL instead of just delete Id for tweet. 

The you can reuse this function for any delete call

Reply:
okay